src/plugins/intel_cpu/src/emitters/plugin/riscv64/jit_eltwise_emitters.cpp

```diff
@@ -682,6 +682,69 @@ void jit_logical_and_emitter::register_table_entries() {
     push_arg_entry_of("one", CONST_1_F);
 }
 
+/// LOGICAL NOT ///
+jit_logical_not_emitter::jit_logical_not_emitter(ov::intel_cpu::riscv64::jit_generator* host,
+                                                 ov::intel_cpu::riscv64::cpu_isa_t host_isa,
+                                                 const std::shared_ptr<ov::Node>& node)
+    : jit_emitter(host, host_isa, get_arithmetic_binary_exec_precision(node)) {
+    prepare_table();
+}
+
+jit_logical_not_emitter::jit_logical_not_emitter(ov::intel_cpu::riscv64::jit_generator* host,
+                                                 ov::intel_cpu::riscv64::cpu_isa_t host_isa,
+                                                 const ov::element::Type exec_prc)
+    : jit_emitter(host, host_isa, exec_prc) {
+    prepare_table();
+}
+
+size_t jit_logical_not_emitter::get_inputs_num() const {
+    return 1;
+}
+size_t jit_logical_not_emitter::aux_fp_gprs_count() const {
+    return 2;
+}
+size_t jit_logical_not_emitter::aux_vecs_count() const {
+    return 1;
+}
+
+void jit_logical_not_emitter::emit_impl(const std::vector<size_t>& in_vec_idxs,
+                                        const std::vector<size_t>& out_vec_idxs) const {
+    if (host_isa_ == ov::intel_cpu::riscv64::cpu_isa_t::gv) {
+        emit_isa<ov::intel_cpu::riscv64::cpu_isa_t::gv>(in_vec_idxs, out_vec_idxs);
+    } else {
+        OPENVINO_THROW("Can't create jit eltwise kernel");
+    }
+}
+
+template <ov::intel_cpu::riscv64::cpu_isa_t isa>
+void jit_logical_not_emitter::emit_isa(const std::vector<size_t>& in_vec_idxs,
+                                       const std::vector<size_t>& out_vec_idxs) const {
+    VReg src0 = VReg(in_vec_idxs[0]);
+    VReg dst = VReg(out_vec_idxs[0]);
+    FReg fzero = FReg(aux_fp_gpr_idxs[0]);
+    FReg fone = FReg(aux_fp_gpr_idxs[1]);
+    load_table_val("one", fone);
+    h->fmv_w_x(fzero, zero);
+    h->vfmv_v_f(dst, fone);
+    switch (exec_prc_) {
+    case ov::element::f32:
+        h->vmfne_vf(mask_vreg(), src0, fzero);
+        h->vfsub_vf(dst, dst, fone, VM::masked);
+        break;
+    default:
+        OV_CPU_JIT_EMITTER_THROW("Unsupported precision");
+    }
+}
+
+std::set<std::vector<element::Type>> jit_logical_not_emitter::get_supported_precisions(
+    const std::shared_ptr<ov::Node>& node) {
+    return {{element::f32, element::f32}};
```

Comment:
Since this op has only one input - there should be only one value in vector with input supported precision. Please fix it